integration_tests/test_pkg_lpdraw.py

```diff
@@ -0,0 +1,34 @@
+from lpdraw import Line, Circle, Clear, Display, DisplayTerminal
+
+from lpython import i32, TypeVar
+from numpy import empty, int32
+
+H = TypeVar("H")
+W = TypeVar("W")
+
+def test_screen(H: i32, W: i32, Screen: i32[H, W]):
+    i: i32
+    j: i32
+    cnt: i32 = 0
+    for i in range(H):
+        for j in range(W):
+            if Screen[i, j] == 255:
+                cnt += 1
+
+    assert cnt == 175
```

Comment:
```suggestion
    assert cnt == -175
```

Reply:
If we do `cnt += (Screen[i, j] - 256):`, I guess the count would be some negative number. Since, for each pixel (not switched on), we add `-256`.

Reply:
Ah! Yes. What I am trying to suggest is to just sum up all the values without any filter. And then make sure that the sum is as expected.

Reply:
Right now, you are ignoring all values `Screen[i, j] != 255` which will not reflect any error for such values in `cnt`. So better to just sum up all the values without any preference.

Reply:
Got it. Done. Thank you so much for the guidance!